src/js/services/user-service.js

```diff
@@ -65,3 +67,30 @@ export function trackEvent(eventName, userProps) {
         return response;
     });
 }
+
+export function updateNowViewing(deviceId) {
+    if (!deviceUpdateThrottle) {
```

Comment:
No, I didn't. There was a portion I didn't quite understand, and it was mixed in with 'isDirty', which I'm already checking in a different way.